apps/legal_archive/commands.py

```diff
@@ -425,6 +425,8 @@ class ImportLegalPublishQueueCommand(superdesk.Command):
     ]
 
     def run(self, page_size=None):
+        if not app.config['LEGAL_ARCHIVE']:
```

Comment:
it won't register it for manage.py, but it is registered also via celery beat and that would still run